### PR TITLE
Create strings.xml

### PR DESCRIPTION
**Pull request by sero583**
Adds german language.

### CODE REVIEW

File: android/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <string name="launch_button">Starte cs16-klient!</string>
+    <string name="cmd_args_text">Kommando linien argumente</string>
+    <string name="cmd_args_text2">Nicht berüheren wenn du nicht weißt warum du das brauchst!</string>
+    <string name="not_found_msg">Nicht gefunden. Prüfe deine APK oder wähle einen anderen Server aus</string>
+    <string name="not_found_title">Irgendetwas ist schief gelaufen...</string>
+    <string name="not_implemented_msg">Dieses Feature ist noch nicht implementiert.</string>
+    <string name="not_implemented_title">Nicht implementiert!</string>
+	<string name="enablezbot">Aktiviere ZBot bots</string>
+	<string name="enableyapb">Aktiviere YaPB bots</string>
+	<string name="defCmdArgs">-konsole -spiel cstrike -bots, u.s.w</string>
```
Comment (a1batross) [inline android/res/values-de/strings.xml]: It's a console arguments. There is no need to translate it.

File: android/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">CS16Client</string>
+    <string name="title_activity_launcher">CS16Client Launcher</string>
+    <string name="launch_button">Starte cs16-klient!</string>
```
Comment (a1batross) [inline android/res/values-de/strings.xml]: klient! I love German, but we are "cs16-client"(or CS16Client sometimes). Client, not Klient.

**Comment by sero583** [comment]
@a1batross So client is for you like a name, not translateable, i thought i could translate everything.
Sorry, my mistake.

**Comment by a1batross** [comment]
It's my fault. I need to mark some string untranslatable. Sorry.

**Comment by sero583** [comment]
@a1batross No problem! :)

**Pull request closed without merging**